app/lib/package/model_properties.dart

```diff
@@ -87,6 +88,16 @@ class Pubspec {
     return map is Map<String, dynamic> ? map : null;
   }
 
+  /// Returns the minimal SDK version for Flutter SDK or Dart SDK, in that order,
+  /// only if specified.
+  ///
+  /// Returns null if the constraint does not follow the `>=<version>` pattern.
+  MinSdkVersion get minSdkVersion {
+    _load();
+    return MinSdkVersion.tryParse('F', _inner.environment['flutter']) ??
+        MinSdkVersion.tryParse('D', _inner.environment['sdk']);
```

Comment:
Didn't we agree that we don't need this?

Reply:
Oh, I may have missed that part of the discussion. I'll remove it for now, it is not that hard to re-add later.